### PR TITLE
Fix rendering images with incorrect aspect ratio

### PR DESCRIPTION
**Pull request by msklv**
Hey guys!

You have developed a great App that can help people.

I fixed the layout a little bit.

### CODE REVIEW

**Review by hellomuthu23 — APPROVED** [review]
Thanks for the fix, highly appreciated!

File: src/styles/styles.css [hunk 1]
```diff
@@ -5,4 +5,5 @@
     --color-background-secondary: #e7edf3;
     --color-warning:orangered;
     --color-border:grey;
+    width: 95%
```
[the remark this answers is not in the document]
Comment (msklv) [reply src/styles/styles.css]: You  fix scrollbar issue better.
This change is not relevant.

**Comment by msklv** [comment]
Updated

File: src/styles/styles.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (hellomuthu23) [reply src/styles/styles.css]: cool, I assumed so!.

**Pull request merged**